### PR TITLE
Add serializers for stock transfer and transfer items

### PR DESCRIPTION
**Pull request by allisonlarson**
* Stock transfer serializer follows convention of making its ‘number’ into its ‘id’ for pushing to wombat
* Stock transfer source location and destination location codes are used
* Transfer item serializes its variants sku
* Specs for formatting

### CODE REVIEW

File: app/serializers/spree/wombat/stock_transfer_serializer.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      def id
+        object.number
+      end
+
+      def source_location
+        object.source_location.code
+      end
+
+      def destination_location
+        object.destination_location.code
```
Comment (athal7) [inline app/serializers/spree/wombat/stock_transfer_serializer.rb]: i think this has the potential to be blank sometimes?

Comment (allisonlarson) [inline app/serializers/spree/wombat/stock_transfer_serializer.rb]: We won't be sending it through the serializer if it's blank though right?

Comment (athal7) [inline app/serializers/spree/wombat/stock_transfer_serializer.rb]: i'm not sure what we might end up doing later. better safe than sorry imo

Comment (allisonlarson) [inline app/serializers/spree/wombat/stock_transfer_serializer.rb]: Wait, are you talking about the destination location itself or the the code field? I can't see a place were destination location would be blank, that is the point of a transfer.. But idk.

**Pull request merged**